project/BRANCHES-AND-TAGS.md

```diff
@@ -0,0 +1,35 @@
+Branches and tags
+=================
+
+Note: details of the release process for the Engine are documented in the
+[RELEASE-CHECKLIST](https://github.com/docker/docker/blob/master/project/RELEASE-CHECKLIST.md).
+
+# Branches
+
+Docker projects should normally have only three living branches at all time, including the regular
+`master` branch:
+
+## `docs` branch
+
+The `docs` branch support documentation updates between product releases. This branch allow us to
+decouple documentation releases from product releases.
+
+## `release` branch
+
+The `release` branch contains the last _released_ version of the code for the project.
+
+The `release` branch is only updated at each public release of the project. The mechanism for this
+is that the release is materialized by a pull request against the `release` branch which lives for
+the duration of the code freeze period. When this pull request is merged, the `release` branch gets
+updated, and its new state is tagged accordingly.
+
+# Tags
+
+Any public release of a compiled binary, with the logical exception of nightly builds, should have
```

Comment:
I think this is silly:
- No one validates GPG signatures unless it's automated
- We don't have a good method for securely distributing these keys or rotating them
- Keys based on individuals mean that they are a lot more prone to being lost, compromised, or the individual stopping being a maintainer, making the set of trusted keys a lot more dynamic and this whole thing even more useless.

If you want trust over every tag, have both @tianon and @jfrazelle tweet them after pushing. That will be a lot more secure. You can even call it twoeet-factor-authentication.